hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/TokensConfig.java

```diff
@@ -49,5 +49,7 @@ public record TokensConfig(
                 boolean nftsUseVirtualMerkle,
         @ConfigProperty(value = "autoCreations.isEnabled", defaultValue = "true") @NetworkProperty
                 boolean autoCreationsIsEnabled,
+        @ConfigProperty(value = "tokens.maxMetadataBytes", defaultValue = "100") @NetworkProperty
```

Comment:
All the configs will have `tokens` appended to them because of line 24 `@ConfigData("tokens")`.  so we don't need `tokens.maxMetadataBytes` here. Just `maxMetadataBytes`